docs/source/UserGuide/GettingStarted/GettingUserIdentity/index.rst

```diff
@@ -53,3 +53,32 @@ If another non-default user group is needed, the command becomes::
   $ dirac-proxy-init -g <user_group>
 
 where ``user_group`` is the desired DIRAC group name for which the user is entitled.
+
+.. note:: Version 8.0 added the possibility to generate proxy with new `dirac-login` command, use *--help* switch for more information. E.g.: dirac-login <user_group>
+
+Token authorization
+-------------------
+
+Starting with the 8.0 version of DIRAC, it is possible to authorize users through third party Identity Providers (IdP),
+such as `EGI Checkin <https://www.egi.eu/services/check-in/>`_ or `WLCG IAM <https://indigo-iam.github.io/v/current/>`_.
+You do not need a certificate for this in a terminal, the main thing is that you must be registered in one of the supported IdP. The registration process is different for each IdP.
+
+Once your account is created, you will be able to register with DIRAC Authorization Server using *--use-diracas* switch of the `dirac-login` command::
+
+  dirac-login <user_group> --use-diracas
+
+You can request to return the access token instead of a proxy using *--token* key::
+
+  dirac-login <user_group> --token
+
+But since not all services currently support tokens, you can get a proxy if you use the *--proxy* key::
+
+  dirac-login <user_group> --proxy --use-diracas
+
+.. note:: if you want to get a proxy after logging in to DIRAC Authorization Server you must first put it in DIRAC, see "Proxy initialization".
```

Comment:
Yes, there is no magic here. As we have agreed, DIRAC will not issue **self-signed** proxies or tokens, so DIRAC must receive them first in order to be able to give them to a user or service (to perform asynchronous tasks on behalf of the user).

**1.**
There are several ways to "upload" a proxy to DIRAC, you know them (+ `dirac-login`). This thing has not changed.
To avoid this, I consider `RCAuth` (in progress), which generates a certificate and a proxy on the fly, but it has significant **limitations**.

**2.**
After that, you can make
```
dirac-login dirac-user --use-diracas
```
 from any installation (not server) without a certificate and after authorization through IAM for example you will receive a proxy.

Or you can configure a client without a certificate (without `dirac-proxy-init`) using `--login` flag (and for this action not need proxy in `ProxyManager`, it can be done with tokens because all involved components running with tornado):
```
dirac-configure --login=https://lbcertifdiracoauth.cern.ch/auth
```

Reply:
For my understanding: is this a workflow that we can consider "advised" or "suggested"? I guess this comes from part of the discussions about proxies -> tokens transition.

Reply:
Sorry for the late answer, as far as I understand the situation around the discussions about the use of tokens and how they are already integrated into other products, I would say that we recommend trying to use tokens, as the transition to tokens is expected in the near future.

I think that in most cases users will prefer to use:
```
dirac-login my-group
```
or `dirac-proxy-init` of course.

Reply:
When interacting with a user, it is important for DIRAC to obtain his **proxy** certificate and **refresh token**. Having received them once, it is expected that they should be enough for about a year. But if there are no problems with receiving tokens: you just need to have a browser and an account in IAM for example, then with a proxy a little harder: you need to have a local certificate to generate proxy.